docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -340,8 +345,9 @@ When you create your domain using the sample provided below, it will obtain the
 from this secret.
 
 A [sample](/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-rcu-credentials/README.md)
-is provided that demonstrates how to create the secret.
-
+is provided that demonstrates how to create the secret.  The schema owner username required will be the
+`schemaPrefix` value followed by an underscore and a component name, such as `FMW1_STB`.  The schema owner
+owner password will be password you provided for regular schema users during RCU creation.
```

Comment:
The schema owner owner password will be password -> The schema owner password will be the password